rgd/geodata/search.py

```diff
@@ -66,6 +67,8 @@ class BoundingBoxSerializer(rfserializers.Serializer):
 
 
 def _add_time_to_query(query, timefield, starttime, endtime, has_created=False):
+    starttime = make_aware(starttime)
+    endtime = make_aware(endtime)
```

Comment:
This could have unintended side effects... perhaps we need to extend the endpoint to accept a DateTime with timezone? or if it does have a timezone, skip this?